artifactory/commands/repository/repository.go

```diff
@@ -665,6 +697,8 @@ var remoteRepoHandlers = map[string]repoHandler{
 	Alpine:    remoteAlpineHandler,
 	Generic:   remoteGenericHandler,
 	Swift:     remoteSwiftHandler,
+	Terraform: localTerraformHandler,
```

Comment:
(let's also create the handler)
```suggestion
	Terraform: remoteTerraformHandler,
```